backend/app/Http/Controllers/Auth/NewPasswordController.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
is it possible to rename this file to PasswordController? and add the function inside of the PasswordResetLinkController file? so that we will have 1 password controller file